src/test/java/com/iabtcf/encoder/field/BooleanEncoderTest.java

```diff
@@ -0,0 +1,27 @@
+package com.iabtcf.encoder.field;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BooleanEncoderTest {
+    private static final String VALUE_0 = "0";
+    private static final String VALUE_1 = "1";
+    private static final String INVALID_VALUE = "231";
+    private static final String EMPTY_VALUE = "";
+
+    @Test
+    public void whenValueIsZeroShouldReturnFalse() {
+        Assert.assertFalse(BooleanEncoder.getInstance().decode(VALUE_0));
+    }
+
+    @Test
+    public void whenValueIsOneShouldReturnTrue() {
+        Assert.assertTrue(BooleanEncoder.getInstance().decode(VALUE_1));
+    }
+
+    @Test
+    public void whenValueIsInvalidShouldReturnFalse() {
```

Comment:
Hi Paulo,
this both lines are actually two test cases, please use separate test cases Empty, invalid). It's easier to find errors if the test covers only one assertion.
Also consider testing null parameter.
Greets
Antje

Reply:
The suggested change was made.